mtags/src/main/scala-3/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -402,7 +402,11 @@ case class ScalaPresentationCompiler(
     ) { access =>
       val driver = access.compiler()
       val uri = params.uri
-      CompilerInterfaces.parseErrors(driver, uri, params.text)
+      val isNotSbt = !uri.getPath.endsWith(".sbt")
+      if (isNotSbt)
+        CompilerInterfaces.parseErrors(driver, uri, params.text)
+      else
+        Nil.asJava
```

Comment:
I've checked scala2 PC and this method has an empty implementation and does nothing.

I'm wondering why we use it?

Reply:
Scala 3 parser diagnostics seem to be nicer, but maybe we can just remove it altogether?

Reply:
I removed them, it should also fix some other issues